tests/ui/run-functional-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+git clone --depth 1 https://github.com/mozilla/addons-server.git
+docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml up -d --build
+until docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml images | grep "addonsserver_addons-frontend_1" ;
+    do printf "."; sleep 1
+done
```

Comment:
You probably want to add `echo` after this so that it prints a new line when it finishes.